sidebars.js

```diff
@@ -1014,6 +1014,18 @@ module.exports = {
                 "self-managed/setup/deploy/amazon/aws-ec2",
               ],
               "Microsoft (Azure)": [
+                {
+                  type: "category",
+                  label: "Azure AKS",
```

Comment:
would suggest to reuse the `Microsoft AKS` one instead of doing `Azure AKS`.

![image](https://github.com/user-attachments/assets/393aec62-1bee-4c7e-ba5a-428bc7772b3c)

While it's not really officially called Mircosoft AKS, Azure AKS is also not a thing as far as I'm aware.

Azure Azure Kubernetes Services.